addons/mail/static/tests/mock_server/mock_models/mail_thread.js

```diff
@@ -425,6 +427,16 @@ export class MailThread extends models.ServerModel {
                 }
             }
         }
+        if (message.partner_ids) {
+            for (const partner_id of message.partner_ids) {
```

Comment:
To better match python, this should only send it to users with inbox enabled in settings.